microprofile.ls/org.eclipse.lsp4mp.ls/src/main/java/org/eclipse/lsp4mp/ls/java/JavaFileTextDocumentService.java

```diff
@@ -358,9 +365,8 @@ private void triggerValidationFor(List<String> uris) {
 	}
 
 	public void propertiesChanged(MicroProfilePropertiesChangeEvent event) {
-		if (documents.propertiesChanged(event)) {
-			triggerValidationForAll(event.getProjectURIs());
-		}
+		documents.propertiesChanged(event);

```

Comment:
Why do we need this change?

Reply:
Ok understood. Which fix do you want to do?

Reply:
A part of https://github.com/eclipse/lsp4mp/issues/162; add a new `MicroProfilePropertiesScope` for config properties, and use that to signal that Java files should be revalidated.

Reply:
hold on ; change I made doesn't work

Reply:
I can't seem to figure out what is broken. I'll try on Windows

Reply:
Some tests fail in Eclipse but not when using Maven on Linux

Reply:
On Windows, `microprofile/projectInfo` seems to be failing for me.

Reply:
JavaTextDocument:142, `getNow()` is used, and this appears to always return `null` when getting the project labels in order to filter the documents to only documents in MicroProfile projects. I don't know how I should go about addressing this.

Reply:
Sorry about that. I figured out what was causing the issue.